### PR TITLE
Resolved default charset warning in jetty-util for #2206

### PR DESCRIPTION
**Pull request by lachlan-roberts**
Ignored warning in B64Code
US-ASCII in UnixCrypt
#2206

Signed-off-by: Lachlan Roberts <lachlan@webtide.com>

### CODE REVIEW

File: jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java [hunk 1]
```diff
@@ -279,6 +280,7 @@ public static String decode(String encoded,String charEncoding)
      * @throws IllegalArgumentException if the input is not a valid
      *         B64 encoding.
      */
+	@SuppressWarnings("DefaultCharset")
```
Comment (joakime) [inline jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java]: tabs vs spaces. :-)
please setup your editor to use spaces.

File: jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java [hunk 2]
```diff
@@ -261,6 +261,7 @@ public static String encode(String s, Charset charEncoding)
...
```
Comment (gregw) [inline jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java]: Need to document why suppress is OK

File: jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java (hunk 1 above)
Comment (joakime) [reply jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java]: This method uses the `Charset` object already in its arguments, why is errorprone flagging it?

Comment (lachlan-roberts) [reply jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java]: `charEncoding` can be null and then the platform default is used

Comment (joakime) [reply jetty-util/src/main/java/org/eclipse/jetty/util/B64Code.java]: Does errorprone honor the `@Nullable` and `@NotNull` type annotations for situations like this?

**Pull request merged**